scripts/packages/VSCodeServer/src/trees.jl

```diff
@@ -126,7 +126,7 @@ function treerender(x::AbstractArray{T,N}) where {T,N}
         if length(x) > MAX_PARTITION_LENGTH
             partition_by_keys(x, sz = MAX_PARTITION_LENGTH)
         else
-            [SubTree(repr(k), wsicon(v), v) for (k, v) in zip(keys(x), vec(x))]
+            collect([SubTree(repr(k), wsicon(v), v) for (k, v) in zip(keys(x), vec(x))])
```

Comment:
Fun fact: Without these commits user types can escape into the JSON writer, where they cause world age issues again.

Reply:
How so?